homeassistant/components/renault/renault_hub.py

```diff
@@ -49,11 +57,16 @@ async def async_initialise(self, config_entry: ConfigEntry) -> None:
 
         self._account = await self._client.get_api_account(account_id)
         vehicles = await self._account.get_vehicles()
+        device_registry = await async_get_registry(self._hass)
```

Comment:
Please use `async_get` instead. That one is not a coroutine function. `async_get_registry` is legacy.